File: _Inventarios/RepCatalogo/ModuloCatalogo.cs

namespace DaxInvent
{
    static class ModuloCatalogo
    {
        public static string ConsultaArt()
        {
            string art = "";
            var bus = new Buscar.frmBuscar();
            art = bus.Buscar(DattCom.datosEmpresa.strConxAdcom, "select Art_Codigo, Art_nombre, Art_unimed   from adcart ", "art_Codigo", "art_nombre", "Consulta", "Buscar Articulo");
            return art;
        }
    }
}

File: _Inventarios/RepCatalogo/ModuloCatalogo.vb
Imports System.Data.SqlClient
Module ModuloCatalogo
    Public Function ConsultaArt() As String
        Dim art As String = ""
        Dim bus As New Buscar.frmBuscar
        art = bus.Buscar(SysEmpDatt.datosEmpresa.strConxAdcom, "select Art_Codigo, Art_nombre, Art_unimed   from adcart ", "art_Codigo", "art_nombre", "Consulta", "Buscar Articulo")
        Return art
    End Function
End Module
